Gems/WhiteBox/Code/Source/Viewport/WhiteBoxPolygonTranslationModifier.cpp

```diff
@@ -289,8 +296,8 @@ namespace WhiteBox
         {
             TranslatePoints(outline, -polygonMidpoint);
         }
-
-        TranslatePoints(triangles, -polygonMidpoint);
+        
+        TranslatePoints(triangles, -polygonMidpoint + normal * cl_whiteBoxPolygonViewOverlapOffset);
```

Comment:
nit: Can we remove the trailing white space please, also I'd move line `286` down to near `300` to keep the variable lifetime as short as it needs to be